website/docs/guides/migration/versions/00-upgrading-to-v1.7.md

```diff
@@ -14,11 +14,38 @@ description: New features and changes in dbt Core v1.7
 
```

Comment:
The release schedule link above should be https://github.com/dbt-labs/dbt-core/issues/8260

Reply:
Good catch! Thank you